ml-agents/mlagents/trainers/upgrade_config.py

```diff
@@ -82,6 +82,34 @@ def remove_nones(config: Dict[Any, Any]) -> Dict[str, Any]:
     return new_config
 
 
+# Take a sampler from the old format and convert to new sampler structure
+def convert_samplers(old_sampler_config: Dict[str, Any]) -> Dict[str, Any]:
+    new_sampler_config: Dict[str, Any] = {}
+    for parameter, parameter_config in old_sampler_config.items():
+        if parameter == "resampling-interval":
+            print(
+                "resampling-interval is no longer necessary for parameter randomization and is being ignored."
+            )
+            continue
+        new_sampler_config[parameter] = {}
+        new_sampler_config[parameter]["sampler_type"] = parameter_config["sampler-type"]
+        if parameter_config["sampler-type"] == "uniform":
```

Comment:
Could we replace these `if` statements with
```suggestion
        new_samp_parameters = dict(parameter_config) # Copy dict
        new_samp_parameters.pop("sampler-type")
        new_sampler_config[parameter]["sampler_parameters"] = new_samp_parameters
```

Reply:
addressed.